src/components/DatePickerDocked/DatePickerDockedHeader.tsx

```diff
@@ -0,0 +1,140 @@
+import { memo, useCallback, useMemo } from 'react';
+import type { ViewStyle } from 'react-native';
+
+import { useComponentStyles, useMolecules } from '../../hooks';
+import type { DisableWeekDaysType } from '../DatePickerInline/dateUtils';
+import DayNames, { dayNamesHeight } from '../DatePickerInline/DayNames';
+
+const buttonContainerHeight = 56;
+const buttonContainerMarginTop = 4;
+const buttonContainerMarginBottom = 8;
+
+export type CalendarHeaderProps = {
+    locale?: string;
+    scrollMode: 'horizontal' | 'vertical';
+    onPrev: () => any;
+    onNext: () => any;
+    disableWeekDays?: DisableWeekDaysType;
+    style?: ViewStyle;
+    year: number;
```

Comment:
derive year and month from date.
drop selectingYear, selectingMonth, selectingYear

Reply:
done @shashank1010